editor/modes/visual-editor/block-list.js

```diff
@@ -106,9 +101,8 @@ export default connect(
 		selectionEnd: getBlockSelectionEnd( state ),
 	} ),
 	( dispatch ) => ( {
-		clearSelectedBlock: () => dispatch( { type: 'CLEAR_SELECTED_BLOCK' } ),
 		onMultiSelect( { start, end } ) {
 			dispatch( { type: 'MULTI_SELECT', start, end } );
 		},
 	} )
-)( clickOutside( VisualEditorBlockList ) );
```

Comment:
Yep we definitely need a decision here. So for now on master, we're considering the browser selection (and focus) as separate from the selectedBlock and multiSelectedBlocks (which should probably be merged into one unique concept).

We're doing so because we want to be able to open the inserter and the sidebar without losing the selection (note that clicking buttons moves the focus to the button).

But multi-selection allows us to delete the selected blocks by hitting "backspace" which has no sense if the selection is not focused (backspace should work on the focused input or text area ...)

also Ella is saying

> The inspector toggle is currently hidden when the multi-select header takes over

This was not the case when this PR was merged.

So should we tie the selection to the focus which means clicking anywhere deselect everything or should we find a middle-ground where we do so only for multi-selection?